src/main/java/io/jenkins/plugins/sample/global_configuration/OnboardingPluginGlobalConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package io.jenkins.plugins.sample.global_configuration;
+
+import hudson.Extension;
+import hudson.Util;
+import hudson.util.FormValidation;
+import hudson.util.QueryParameterMap;
+import jenkins.model.GlobalConfiguration;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.QueryParameter;
+
+
+@Extension
+@Symbol("OnboardingPlugin")
+public class OnboardingPluginGlobalConfiguration extends GlobalConfiguration {
```

Comment:
https://github.com/vwagh-dev/demo-plugin/pull/4/files#r1675753069

Reply:
Taken care of respective comment.